tests/test_claude_wrapper_hooks.py

```
@@ -178,11 +178,21 @@ def test_stale_socket_skips_hook_injection(failures: list[str]) -> None:
     expect(claudecode == "__UNSET__", f"stale socket: expected CLAUDECODE unset, got {claudecode!r}", failures)
 
 
+def test_print_flag_skips_hook_injection(failures: list[str]) -> None:
+    """Non-interactive flags (-p/--print) should bypass hooks entirely.  See #2206."""
+    for flag, label in [("-p", "-p"), ("--print", "--print"), ("--output-format", "--output-format"), ("--print-auth-token", "--print-auth-token")]:
+        argv = [flag, ".", "--model", "haiku"] if flag in ("-p", "--print") else [flag]
+        code, real_argv, cmux_log, stderr, claudecode = run_wrapper(socket_state="live", argv=argv)
```

Ruff (0.15.7)

[warning] Unpacked variable `cmux_log` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)